app/containers/ItunesAppContainer/index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ *
+ * ItunesAppContainer
+ *
+ */
+
+import React, { useEffect, memo, useState } from 'react';
```

Comment:
so the import order that we follow is 

1. third-party-imports
2. aliases
3. relative paths